index.js

```diff
@@ -0,0 +1,23 @@
+var prompt = require('prompt');
+var fs = require('fs');
+
+prompt.start();
+
+prompt.get(['bitcoin_address', 'paypal_me', 'analytics'], function (err, result) {
+  fs.readFile( __dirname + '/index.html', function (err, data) {
```

Comment:
Leave the original README file with my information (i.e. paypal.me/k4m4 & bitcoin:13yNEtgGjkit7p2JciKz8vH7U2yHbDo8yh) and move the index.html file to the assets folder. (`/index.html` --> `/assets/index.html`)